addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2593,7 +2593,7 @@ export class Wysiwyg extends Component {
                 description: _t('Insert an image'),
                 fontawesome: 'fa-file-image-o',
                 callback: () => {
-                    this.openMediaDialog();
+                    this.openMediaDialog({ noVideos: !editorOptions.allowCommandVideo });
```

Comment:
The fix is not at the correct place. It should be in `openMediaDialog` itself.
Given a scenario:
1. Insert an image. (you don't see video tab in media dialog) :)
2. Double click on image or use replace in toolbar. (you see the video tab in media dialog) :(

Reply:
Ah my bad! I've updated the code, thanks!